### PR TITLE
Add generic Image component and high five image component

### PR DESCRIPTION
**Pull request by mayuriardad**
- Add image core component
- Add high five image core component
- Move assets/images directory to public

### CODE REVIEW

File: react-frontend/src/coreComponents/HignFiveImageComponent.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+HighFiveImageComponent.defaultProps = {
+  className: "",
+  fluid: false,
+  rounded: false,
+  roundedCircle: false,
+  thumbnail: false,
+  size: "md",
+};
+
+export default HighFiveImageComponent;
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/HignFiveImageComponent.js:47]: Wrap exported component in `React.memo`.
i.e. `export default React.memo(HighFiveImageComponent)`

File: react-frontend/src/coreComponents/HignFiveImageComponent.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  className: PropTypes.string,
+  fluid: PropTypes.bool,
+  rounded: PropTypes.bool,
+  roundedCircle: PropTypes.bool,
+  thumbnail: PropTypes.bool,
+  size: PropTypes.oneOf(["md", "lg"]),
+};
+
+HighFiveImageComponent.defaultProps = {
+  className: "",
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/HignFiveImageComponent.js:39]: Bad idea to have `""` (empty quoutes) as default value

File: react-frontend/src/coreComponents/HignFiveImageComponent.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  fluid: false,
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/HignFiveImageComponent.js:40]: Make this `true` by default

File: react-frontend/src/constants/appConstants.js [hunk 4]
```diff
@@ -1 +1,3 @@
 export const REDIRECT_TIMEOUT = 3000;
+
+export const IMAGE_ASSET_PATH = `${process.env.PUBLIC_URL}/assets/images/`;
```
Comment (sahilbhatia) [inline react-frontend/src/constants/appConstants.js:3]: How about renaming `IMAGE_ASSET_PATH` to `IMG_BASE_PATH`?

File: react-frontend/src/coreComponents/ImageComponent.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+ImageComponent.defaultProps = {
+  className: "",
+  fluid: false,
+  rounded: false,
+  roundedCircle: false,
+  thumbnail: false,
+};
+
+export default ImageComponent;
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/ImageComponent.js:40]: Wrap exported component in `React.memo`.
i.e. `export default React.memo(ImageComponent)`

File: react-frontend/src/coreComponents/ImageComponent.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  src: PropTypes.string.isRequired,
+  className: PropTypes.string,
+  fluid: PropTypes.bool,
+  rounded: PropTypes.bool,
+  roundedCircle: PropTypes.bool,
+  thumbnail: PropTypes.bool,
+};
+
+ImageComponent.defaultProps = {
+  className: "",
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/ImageComponent.js:33]: Bad idea to have `""` (empty quoutes) as default value

File: react-frontend/src/coreComponents/ImageComponent.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  fluid: false,
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/ImageComponent.js:34]: Make this `true` by default

File: react-frontend/src/coreComponents/HignFiveImageComponent.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+  />
+);
+
+HighFiveImageComponent.propTypes = {
+  className: PropTypes.string,
+  fluid: PropTypes.bool,
+  rounded: PropTypes.bool,
+  roundedCircle: PropTypes.bool,
+  thumbnail: PropTypes.bool,
+  size: PropTypes.oneOf(["md", "lg"]),
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/HignFiveImageComponent.js:35]: Does `oneOf` cover `required`?

File: react-frontend/src/coreComponents/HignFiveImageComponent.js [hunk 9]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
```
Comment (sahilbhatia) [inline react-frontend/src/coreComponents/HignFiveImageComponent.js:1]: Remove `Image` from filename. i.e. `HighFiveComponent`

File: react-frontend/src/coreComponents/HignFiveImageComponent.js (hunk 8 above)
Comment (mayuriardad) [reply react-frontend/src/coreComponents/HignFiveImageComponent.js:35]: no, thats why kept default value for the prop as "md"

**Pull request merged**